### DISCUSSION TITLE
keyball39 のトラックボールが反応しない

### DISCUSSION THREAD

**Discussion by puzzlemeter** [Yowkees/keyball#254] [Q&A] [answered]
お世話になります。

先日、トラックボールセンサーを壊し、センサーだけ再注文をお願いした者です。

センサー届いてから、差し込んだのち、両側の pro micro をそれぞれファームを書き込みしてみましたが、相変わらずトラックボールが動きませんでした。

さらに、フラックスを amazon で購入し、周りを塗った上、ブリッジを直してみましたが、変化が見られないため、
ハンダ付けが微妙なところが他にないか、写真でご確認をお願いできれば助かります。

![keybal39](https://github.com/Yowkees/keyball/assets/8602939/b1cd3844-9711-44f8-af08-b27e911e4779)
![keyball39-2](https://github.com/Yowkees/keyball/assets/8602939/ec29d86e-0a83-4e3a-9360-c701ca65538b)
![keyball39-3](https://github.com/Yowkees/keyball/assets/8602939/4f65ee36-49d0-44f4-9d5c-8da54c1fbd90)

補足情報として、全 key と oled は正常動作している状態です。
led は気分転換でいくつか付けてみましたが、今のところ、1,2 しか光りません。（LEDは別途ゆっくり調査する予定なので、あくまでも参考情報として書き込みました。LED が動かなくても気にしていません。）

何卒よろしくお願いします。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。
センサー基板交換後ということでミドル基板に問題があることで間違いないと思います。

光の加減かも知れませんが下記写真の矢印の端子がうまくピンと接合していないように見えますので半田を足してみるととどうでしょうか？

また、L字7Pコンスルーに半田が付着しているように見えます。もしかするとコンスルーピンの柔軟性が失われて、トラックボール基板に接続不良しているかもしれませんので、この辺りの導通も確認いただければと思います。

自力での修復が不可能そうでしたら、その後の修正方法をご相談したいので、白銀ラボウェブサイトのお問合せよりメッセージいただければ幸いです。
以上、お手数ですがご対応お願いいたします。
![kb39_trackballfix](https://github.com/Yowkees/keyball/assets/84209772/48b8d7d3-321c-4581-80a2-12140c8aa74e)

**Reply by puzzlemeter** [reply]
ご指摘のとおり、矢印の箇所と周辺にもハンダを上乗せしたところ、動くようになりました。

ありがとうございます！

やはり自分の目というか、思い込みは信用できませんでした〜
本当に助かりました orz

**Reply by Yowkees** [reply]
無事に動作したようで良かったです！安心しました。
ぜひ末永くご愛用いただければ幸いです。
